File: Java/Processor.java
public class Processor {

    /* Private Attribute */
    private String name;
    private int price;

    /* Constuctor */
    public Processor() {
        
    }

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    /* Setter dan Getter Name */
    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return this.name;
    }

    /* Setter dan Getter Price */
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

}

File: Python/Processor.py
class Processor():

    #private attributes
    __name = "Processsor Name"
    __price = 0

    #Constructor
    def __init__(self, name = "Processor Name", price = 0):
        self.__name = name
        self.__price = price

    #Setter dan Getter Name
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name
    
    #Setter dan Getter Price
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price